### DISCUSSION TITLE
Unable to generate Prisma client in Nuxt

### DISCUSSION THREAD

**Discussion by AQian0** [prisma/prisma#25529] [Q&A]
### Bug description

I use Nuxt Module for Prisma.

After running the Prisma CLI, it generates the prisma schema file and database migrations folder normally. however, it stays at the step of generating the prisma client.

<img width="1548" alt="image" src="https://github.com/user-attachments/assets/1758f587-c765-47a6-9ce7-c5a3d9a81df2">

### How to reproduce

1. Create a Nuxt project
2. Run 'bun i'
3. Run 'bun nuxi@latest module add @prisma/nuxt'

### Expected behavior

It should work as shown in the official documentation.

### Prisma information

Not yet generated, or consistent with official documentation.

### Environment & setup

- OS: MacOS Sequoia 15.0.1
- Database: Not connected yet, probably using its own SQLite
- Bun version: 1.1.33

### Prisma Version

Not yet successfully installed

**Comment by ludralph** [comment]
Hi @AQian-Cup 

This issue has been reported [here](https://github.com/prisma/nuxt-prisma/issues/31). You may have to use npm for now instead of bun until this issue is fixed.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
